devops/charts/bg-monitoring/templates/app/service.yaml

```diff
@@ -1,12 +1,26 @@
+{{ if .Values.app.enabled }}
+---
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Chart.Name }}-app
+{{- with .Values.app.serviceAnnotations }}
+   annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
+  name: {{ template "bg-monitoring.fullname" . }}-app
+  labels:
+{{ include "bg-monitoring.labels" . | indent 4 }}
+    app.kubernetes.io/component: app
+    app: bg-monitoring
 spec:
-  selector:
-    deploymentconfig: {{ .Chart.Name }}-app
+  type: ClusterIP
   ports:
-    - name: 8080-tcp
+    - name: http
```

Comment:
I think the new name is misleading because http is port 80.

Reply:
yep but for Tomcat default port to run http is 8080. So anyway if it really need we can change  the name to appropriate name.

Reply:
Let's change name to "default".

Reply:
updated